### PR TITLE
db-refactor-12

### PR DESCRIPTION
**Pull request by freesig**
- Removes local sync and other agent specific calls from gossip.

- Depends on #1128 
- Summary in #1130

### CODE REVIEW

File: crates/kitsune_p2p/direct/src/v1.rs [hunk 1]
```diff
@@ -899,55 +897,56 @@ async fn handle_query_op_hashes(
...
-        entries.extend(es.into_iter());
-    }
-
-    let mut entries: Vec<_> = entries
-        .into_iter()
-        .map(|e| e.hash().clone().to_kitsune_op_hash())
-        .collect();
-    entries.sort();
-    entries.dedup();
+    // let agents = todo!();
```
Comment (thedavidmeister) [inline crates/kitsune_p2p/direct/src/v1.rs:900]: is this intentionally left commented out?

File: crates/kitsune_p2p/direct/src/v1.rs [hunk 2]
```diff
@@ -899,55 +897,56 @@ async fn handle_query_op_hashes(
...
-                .get_entry(root.clone(), agent.clone(), hash)
-                .await
-            {
-                out.push((op_hash.clone(), entry.as_wire_data_ref().to_vec()));
-            }
-        }
-    }
-
-    Ok(out)
+    // let FetchOpDataEvt {
```
Comment (thedavidmeister) [inline crates/kitsune_p2p/direct/src/v1.rs:925]: more commented code

File: crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/bloom.rs [hunk 3]
```diff
@@ -140,7 +129,7 @@ impl ShardedGossipLocal {
     /// - The expected performance per op is average 10ms and worst 100 ms.
     pub(super) async fn check_ops_bloom(
         &self,
-        local_agents_within_arc_set: Vec<(Arc<KitsuneAgent>, ArcInterval)>,
+        // local_agents_within_arc_set: Vec<(Arc<KitsuneAgent>, ArcInterval)>,
```
Comment (thedavidmeister) [inline crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/bloom.rs:132]: commented code

File: crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs [hunk 4]
```diff
@@ -264,13 +282,14 @@ async fn test_three_way_sharded_ownership() {
     let get_op_hashes = |a: usize| {
         let evt_sender = evt_sender.clone();
         let space = space.clone();
+        let arc = agent_arcs[a].1.clone();
+        dbg!(&arc);
```
Comment (thedavidmeister) [inline crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs:286]: dbg statement

File: crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs [hunk 5]
```diff
@@ -543,22 +543,23 @@ impl SwitchboardState {
...
-                                        .ops
-                                        // Does agent hold this op?
-                                        .get(op_loc8)
-                                        // Does it meet the limbo criteria of the query?
-                                        .map(|op| include_limbo || op.is_integrated)
-                                })
-                                .unwrap_or(false)
-                        })
+                        && arc_set.contains((**op_loc8).into())
+                // && agents.iter().fold(false, |yes, (agent, arc_set)| {
```
Comment (thedavidmeister) [inline crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs:547]: commented code

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

**Review by neonphog — APPROVED** [review]

File: crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs (hunk 5 above)
Comment (freesig) [reply crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs:547]: Above all fixed in `140576f89e08d81a09e9c21a2a84d9d17a66482b`

File: crates/kitsune_p2p/direct/src/v1.rs (hunk 1 above)
Comment (freesig) [reply crates/kitsune_p2p/direct/src/v1.rs:900]: Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b

File: crates/kitsune_p2p/direct/src/v1.rs (hunk 2 above)
Comment (freesig) [reply crates/kitsune_p2p/direct/src/v1.rs:925]: Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b

File: crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs (hunk 4 above)
Comment (freesig) [reply crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip/tests/handler_builder.rs:286]: Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b

File: crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs (hunk 5 above)
Comment (freesig) [reply crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_state.rs:547]: Fixed in https://github.com/holochain/holochain/commit/140576f89e08d81a09e9c21a2a84d9d17a66482b

**Pull request closed without merging**